launch_ros/launch_ros/actions/composable_node_container.py

```diff
@@ -34,8 +35,10 @@ class ComposableNodeContainer(Node):
     def __init__(
         self,
         *,
-        node_name: SomeSubstitutionsType,
-        node_namespace: SomeSubstitutionsType,
+        name: SomeSubstitutionsType = None,
+        namespace: SomeSubstitutionsType = None,
```

Comment:
@jacobperron nit: if these have `None` as default, type annotation should read `Optional[SomeSubstitutionsType]`

Reply:
Fixed 7c00297